elementary/monitor/debug.py

```diff
@@ -21,7 +21,7 @@ def run(self) -> bool:
             dbt_runner.run_operation("test_conn", quiet=True)
         except DbtCommandError as err:
             click.echo(
-                f"Could not connect to the Elementary db and schema. See details below\n\n{err}"
+                f"Could not connect to the Elementary db and schema. See details below\n\n{err.get_exception_message() or err}"
```

Comment:
Thinking out loud, maybe we should just make `get_exception_message()` return `str(self)` if it fails rather than `None`?